test/evptests.txt

```diff
@@ -2,6 +2,63 @@
 #aadcipher:key:iv:plaintext:ciphertext:aad:tag:0/1(decrypt/encrypt)
 #digest:::input:output
 
+# BLAKE2 tests, using same inputs as MD5
```

Comment:
On Wed, Mar 9, 2016 at 3:24 PM, ekasper <notifications@github.com> wrote:

> In test/evptests.txt
> <https://github.com/openssl/openssl/pull/566#discussion_r55609025>:
>
> > @@ -2,6 +2,63 @@
> >  #aadcipher:key:iv:plaintext:ciphertext:aad:tag:0/1(decrypt/encrypt)
> >  #digest:::input:output
> >
> > +# BLAKE2 tests, using same inputs as MD5
>
> How were these generated? Are these official / do we have any official
> test vectors?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openssl/openssl/pull/566/files#r55609025>.
>
I generated them with the official reference source.  All of the official
test vectors are done with key != 0, and since we always set key == 0,
there are no applicable official vectors.